linchpin/cli/__init__.py

```diff
@@ -183,7 +183,7 @@ def _write_distilled_context(self, run_data):
         This method takes all of the provided run_data, loops through the
         distiller section of the linchpin.constants and writes out the
         linchpin.context (name TBD) to the 'resources' directory.
-
+    
```

Comment:
flake8 failed on these. The extra line is fine, but whitespace on a line by itself is not.